.github/actions/deploy-template-operator-with-modulereleasemeta/action.yml

```diff
@@ -19,7 +19,7 @@ runs:
       shell: bash
       run: |
         make build-manifests
-        kyma alpha create module --module-config-file ./module-config.yaml --path . --registry localhost:5111 --insecure
+        module create --module-config-file ./module-config.yaml --registry http://localhost:5111 --insecure
```

Comment:
We need to modify the module-config.yaml in the template-operator to have the defaultCR and manifest from the release URLs instead of the local file paths.

Reply:
This will be done when we bump the modulectl version here since we already deprecated the use of local file paths